### ISSUE TITLE
Missed defaults in MSendMsg

### ISSUE THREAD

**Issue by stanch** [dzsessona/scamandrill#3]
I think these can be `Option[String] = None` (they are also stated as optional in the scaladoc).

```scala
...
val bcc_address: String,
val tracking_domain: String,
val signing_domain: String,
val return_path_domain: String,
...
```

**Comment by nlochschmidt** [comment]
I opened a pull request for this. Btw. what is the reason for MSendMessage being a normal `class` instead of a `case class`?

**Comment by dzsessona** [comment]
Hello Niklas , thank you for your work. To be honest I didn't have time yet
to look at it but will do it shortly and will answer your questions.

Thanks again, Diego.

**Comment by dzsessona** [comment]
fixed by f7fe3b5710a14c07ff844e5e8dfb4a76e6f556a2
